modules/odc-importing-codebase-from-git-to-create-application.adoc

```diff
@@ -11,7 +11,7 @@ The following procedure walks you through the *From Git* option in the *Develope
 
 .Procedure
 
-. In the *+Add* view, click *From Git* to see the *Import from git* form.
+. In the *+Add* view, visit the *Git Repository* tile and click on the *From Git* to see the *Import from git* form.
```

Comment:
'visit' is not the right word in this context.
Suggest: In the *+Add* view, click *From Git* in the *Git Repository* tile to see the *Import from git* form.

Reply:
Ideally we use click **button/option**, and not click on the button/option, unless you want to mention the type of field like, for example, click the **YAML** tab

Reply:
Done!